kivy/core/camera/camera_opencv.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, **kwargs):
 
     def init_camera(self):
         # consts have changed locations between versions 2 and 3
-        if self.opencvMajorVersion == 3:
+        if self.opencvMajorVersion == 3 or self.opencvMajorVersion == 4:
```

Comment:
Would you mind changing both of these to `if self.opencvMajorVersion in (3, 4)` etc., just for code neatness?